fastlane.gemspec

```diff
@@ -57,4 +57,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock', '~> 1.19.0'
   spec.add_development_dependency 'coveralls'
   spec.add_development_dependency 'rubocop', '~> 0.29'
+  spec.add_development_dependency 'appium_lib', '~> 4.1.0'
```

Comment:
Please don't add this as a dependency, but add the following line to the top of your `self.run` method instead:

```ruby
Actions.verify_gem!('appium_lib')
```

This way the dependency doesn't get installed for every user, but only for people who actually want to use this action :+1: